### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mikebrow**
make a bold notice regarding where people should go for guidance

### CODE REVIEW

**Review by jdolitsky — CHANGES_REQUESTED** [review]
IMO the README is still misleading because it is so long. Can we create a file called README_ARCHIVED.md to maintain all this stuff, then do something like i suggest here? https://github.com/opencontainers/tob/issues/126#issuecomment-1634942682

**Comment by mikebrow** [comment]
> IMO the README is still misleading because it is so long. Can we create a file called README_ARCHIVED.md to maintain all this stuff, then do something like i suggest here? [opencontainers/tob#126 (comment)](https://github.com/opencontainers/tob/issues/126#issuecomment-1634942682)

Appreciate the perspective.. Let's treat this (modifying this repo's readme with a quick succession of prs) as a work in progress moving in that ^ direction, cutting out the parts that don't need to be here (should not be here) as those parts show up in the 1.1 specs and probably including a few more tabs (here) than you have suggested just to preserve external links for redirection to the new suggested spec doc sub tabs in said artifact author guidance as it gets merged in. No reason to merge pointing to links that are not there yet :-)

**Review by jdolitsky — APPROVED** [review]
LGTM

**Review by vbatts — APPROVED** [review]

**Review by dmcgowan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,82 +1,27 @@
...
-More particularly this repository has been tasked by the [OCI TOB](https://github.com/opencontainers/tob/blob/master/proposals/artifacts.md) to serve 3 primary goals:
-
-1. **artifact authors** - [guidance for authoring new artifact types.][artifact-authors] Including a clearing house for well known artifact types.
-1. **registry operators and vendors** - guidance for how operators and vendors can support new artifact types, including how they can opt-in or out of well known artifact types. Registry operators that already implement `media-type` filtering will not have to change. The artifact repo will provide context on how new `media-type`s can be used, and how `media-type`s can be associated with a type of artifact.
-1. **clearing house for well known artifacts** - artifact authors can submit their artifact definitions, providing registry operators a list by which they can easily support.
-
-By providing an OCI artifact definition, the community can continue to innovate, focusing on new artifact types without having to build yet another storage solution (YASS).
+## Mission Update
```
Comment (SteveLasker) [inline README.md:8]: the act of archiving, and directing users to the new reopos is a bit redundant to. See proposed changes in #70

**Review by SteveLasker — CHANGES_REQUESTED** [review]
See PR #70 for some additional wording. 
**Note:** this also depends on https://github.com/opencontainers/image-spec/pull/1095 for the artifacts-guidance.md link

**Comment by mikebrow** [comment]
merged into #70 for additional context

**Pull request merged**